core-web/libs/template-builder/src/lib/components/template-builder/template-builder.component.ts

```diff
@@ -250,7 +250,7 @@ export class TemplateBuilderComponent implements OnInit, AfterViewInit, OnDestro
                 opacity: '1'
             };
             this.cd.detectChanges();
-        }, 250);
+        }, 350);
```

Comment:
GridStack does a weird animation that is not really good in UX terms. So this delay prevents the user to watch it.

I added 100ms because the templates portlet was the only one were you could see this animation. It's not affecting any logic/rendering at all, is just a UX matter